SU-SWS/saa_victoria_subtheme
# Adapt 286  more button additions

# READY FOR REVIEW

# Summary
- Adds focus blue box shadow on buttons
- remove the inclusion of full version of decanter, i think this was problematic for overriding decanter styles
- Adds mixin for cta white on bright red background
- Adds bright red button mixin
* i will use these mixins in my other branch so no real need to test much.

# Review By (Date)
- 6/10 if possible to continue banner work, or 6/11

# Urgency
- necessary to continue banner styles

# Steps to Test

1. pull down the branch
1. see the buttons look fine, inspect and add ::focus and see blue drop shadow
2. other mixins will be applied in the banner styles branch

# Affected Projects or Products
- n/a

# Associated Issues and/or People
- ADAPT-286 and ADAPT-367 and ADAPT-356 because they'll use the mixins for the new bright_red button.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
